defaults/main.yml

```diff
@@ -18,3 +17,27 @@ i2_include_plugins:
   - "nscp"
 i2_custom_constants: {}
 i2_custom_features: {}
+i2_custom_templates: {}
+i2_custom_commands: {}
+i2_remove_unmanaged_features: false
+
+i2_master: no
+i2_satellite: no
+i2_node_parent: master
```

Comment:
Yes, the default would be for a node to become an agent in a simple master-agent set-up.
Inside `host-` or `group_vars`, one would override these defaults.
E.g. a master would have:

```yaml
i2_master: yes
i2_custom_features:
  ApiListener:
    api:
      …
```

And an agent connecting not to the master but to a satellite (or rather the `group_vars` file) would have 

```yaml
---
i2_node_parent: satellitenodename
```

I realize all of this would be easier to understand if this commit came with tests and extended documentation.